### ISSUE TITLE
Menu: Esc key should close the menu

### ISSUE THREAD

**Issue by eddysims** [GetJobber/atlantis#696]
The `Esc` key does not work as advertised (VoiceOver, for example, understands that Menu is a menu and reads “press the Esc key to close”, but we do not offer this functionality). This means that to close the menu, the user must ensure they have tabbed to the button and then hit return/space to close.
